.github/workflows/package.yml

```
@@ -327,8 +327,14 @@ jobs:
       # Test
       - name: Test
         run: |
+          if [[ "${{ matrix.test-docker-image }}" == arm64* ]]; then
+            echo "TEST_DOCKER_PLATFORM=linux/arm64" >> $GITHUB_ENV
+          else
+            echo "TEST_DOCKER_PLATFORM=linux/amd64" >> $GITHUB_ENV
+          fi
           docker run \
             --rm \
             --volume ${PWD}:/groonga:ro \
+            --platform ${{ env.TEST_DOCKER_PLATFORM }} \
```

これは動かないんじゃないかな。`$GITHUB_ENV`に入れたやつは次のステップからじゃないと参照できなそう。

シンプルにローカル変数で十分そう。

```suggestion
          case "${{ matrix.test-docker-image }}" in
            arm64v8/*)
              platform=linux/arm64
              ;;
            *)
              platform=linux/amd64
              ;;
          esac
          docker run \
            --platform ${platform} \
            --rm \
            --volume ${PWD}:/groonga:ro \
```

---

ちゃんと把握できていなかったです。ありがとうございますmm
https://docs.github.com/ja/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable

---

↑の内容を適用しました。
https://github.com/groonga/groonga/actions/runs/9710461996 でCI実行中です。